db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveOvirtUseApiFromAttr < ActiveRecord::Migration[6.0]
+  def change
+    compute_resources = ComputeResource.all.unscoped.where(type: 'Foreman::Model::Ovirt')
+    compute_resources.each do |ovirt_resource|
+      ovirt_resource.attrs = ovirt_resource.attrs.except(:ovirt_use_v4)
```

Comment:
not important but you could also do:
```suggestion
      ovirt_resource.attrs.except!(:ovirt_use_v4)
```